Makefile

```diff
@@ -15,8 +15,8 @@ test:
 	python -m coverage run --source pytorch_lightning -m pytest pytorch_lightning tests pl_examples -v --flake8
 	python -m coverage report
 
-	# specific file
-	# python -m coverage run --source pytorch_lightning -m pytest --flake8 --durations=0 -v -k
+	# example for specific file/test
```

Comment:
I think that we do not need it here makefile anymore...
just if it is missing add it to the contribution guide

Reply:
Done! There's already a mention of how to do this in the section for adding a new test, but I've also added a concrete example for an existing test.